tests/test_study/test_study_examples.py

```diff
@@ -0,0 +1,49 @@
+from openml.testing import TestBase
+
+
+class TestStudyFunctions(TestBase):
+    _multiprocess_can_split_ = True
+    """Test the example code of Bischl et al. (2018)"""
+
+    def test_Figure1a(self):
+        """Test listing in Figure 1a on a single task and the old OpenML100 study
+
+        import openml
```

Comment:
you sure want to copy the code into the comment?

Reply:
That's the original code from the listing which is a bit different from the actual code of the unit test. So yes, I want this code there. I'll add a sentence about this.